Samples/Swift/DaysUntilBirthday/DaysUntilBirthdayUITests(iOS)/DaysUntilBirthdayUITests_iOS.swift

```diff
@@ -68,6 +69,23 @@ class DaysUntilBirthdayUITests_iOS: XCTestCase {
       return XCTFail("Signing out should return user to sign in view")
     }
   }
+
+  func testFetchVerificationSignalAndDisconnect() {
+    sampleApp.launch()
+
+    XCTAssertTrue(signIn())
```

Comment:
Is this needed? Maybe the sample app has the sign in screen first? I think that's the case...anyway, is it possible to test this flow without having a sign in first? That would get things closer to the change in this pr.

Reply:
One way to accomplish this is to have a verify button under the sign in button so that the verify flow can be tested without a sign in first.